app/models/in_person_enrollment.rb

```diff
@@ -126,6 +127,16 @@ def days_to_due_date
     (today...due_date).count
   end
 
+  def on_notification_sent_at_updated
+    if self.notification_sent_at && self.notification_phone_configuration
+      self.notification_phone_configuration.destroy
+    end
+  end
```

Comment:
@dawei-nava  You might want to check in with @night-jellyfish. I think delete might be handled in her issue?  Check in with her maybe, I could be wrong. See her [PR](https://github.com/18F/identity-idp/compare/main...brittany/lg-10167-delete-phone-no-when-enroll-cancelled)

Reply:
@gina-yamada Only the cancellation is clearly handled in [LG-10167](https://cm-jira.usa.gov/browse/LG-10167) / https://github.com/18F/identity-idp/compare/main...brittany/lg-10167-delete-phone-no-when-enroll-cancelled; this or another ticket still needs to delete the phone number for other status changes.